pipreqs/pipreqs.py

```diff
@@ -115,16 +116,22 @@ def filter_line(l):
     return len(l) > 0 and l[0] != "#"
 
 
-def generate_requirements_file(path, imports):
+def generate_requirements_file(path, imports, omit_version=None):
     with open(path, "w") as out_file:
         logging.debug('Writing {num} requirements: {imports} to {file}'.format(
             num=len(imports),
             file=path,
             imports=", ".join([x['name'] for x in imports])
         ))
-        fmt = '{name}=={version}'
-        out_file.write('\n'.join(fmt.format(**item) if item['version'] else '{name}'.format(**item)
-                                 for item in imports) + '\n')
+
+        if omit_version is True:
```

Comment:
Stylistic note: this can simply be `if omit_version:`, since `True` and `False` are the only possible values here (enforced by `docopt`).